src/osha/oira/services/country.py

```diff
@@ -0,0 +1,49 @@
+from plone import api
+from plone.restapi.services import Service
+
+
+class Tools(Service):
+    """A JSON service version of @@country-tools"""
+
+    def reply(self):
+        tree = []
+        ctv = api.content.get_view("country-tools", self.context, self.request)
+        country = self.context.getId()
+        for sector in ctv.sectors:
+            groups = []
+            sector_path = "/".join([country, sector["id"]])
```

Comment:
I would use an fstring:
```suggestion
            sector_path = f"{country}/{sector['id']}"
```